src/main/resources/application.properties

```diff
@@ -53,38 +49,38 @@ spring.data.mongodb.database: eiffel_intelligence
 # spring.data.mongodb.username:
 # spring.data.mongodb.password:
 # Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
-server.session-timeout: 1200
+server.session.timeout: 1200
 # Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
 # Name of the database collection where aggregated objects are stored
-aggregated.collection.name: aggregated_objects
+aggregations.collection.name: aggregations
 # Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
 # using the example rule set 'AllEventsRules' to avoid the storage growing too large.
 # Recommended setting is 10 minutes.
-aggregated.collection.ttlValue:
+aggregations.collection.ttl:
 # The name of the database collection where to store the mapping between
 # aggregated objects and id of the Eiffel events that contributed to those
-event_object_map.collection.name: event_object_map
+event.object.map.collection.name: event_object_map
 # Name of the database collection where subscriptions are stored
-subscription.collection.name: subscription
+subscriptions.collection.name: subscriptions
 # Name of the database collection where we store pair of 'subscription:aggregated object'
 # where notification has been sent and should not be repeated
-subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
+subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
 # The waitlist contains Eiffel events which are waiting to be aggregated
-# Configure the database collection name for the waitlist
+# Configure the database collection name for the wait list
 waitlist.collection.name: wait_list
 # Set the time-to-live value for Eiffel events in the waitlist database (seconds)
-waitlist.collection.ttlValue: 600
+waitlist.collection.ttl: 600
 # Resend information for unprocessed events in wait list, time in milliseconds
-waitlist.initialDelayResend: 2000
-waitlist.fixedRateResend: 15000
+waitlist.resend.initial.delay: 2000
+waitlist.resend.fixed.rate: 15000
 # Name of the collection where failed notifications are stored
-failed.notification.collection-name: failed_notifications
-# Number of retries if a notification fail
-notification.failAttempt: 3
+failed.notifications.collection.name: failed_notifications
 # Configure time-to-live for failed notifications in database (seconds)
-notification.ttl.value: 600
+failed.notifications.collection.ttl: 600
+# Number of retries if a notification fail
+notification.retry: 3
```

Comment:
Should this be plural also? Because this configuration goes for all notifications

Reply:
Do you mean notifications.retry or notification.retries?

Reply:
I was referring to having notifications in plural since it's changed to plural in the other properties but maybe it sounds wrong to have "notifications.retry"?

Reply:
I don't know. It sounds a bit strange to my ear but then again properties are not actual sentences. What do you think?

Reply:
Keep it as notification.retry then, if nobody else complains about it 😄